eclipse-scout-core/src/tree/TreeAdapter.ts

```diff
@@ -319,6 +335,18 @@ export class TreeAdapter extends ModelAdapter {
     objects.replacePrototypeFunction(Tree, '_createTreeNode', TreeAdapter._createTreeNodeRemote, true);
     objects.replacePrototypeFunction(Tree, '_updateMarkChildrenChecked', TreeAdapter._updateMarkChildrenCheckedRemote, true);
   }
+
+  static isRemote(node: AdapterTreeNode, includeHybrid?: boolean): boolean {
```

Comment:
All usages seem to pass `true` as second argument, maybe set that as the default value? If the intention was to force the caller to think about whether they want hybrid nodes to be treated as remote nodes, they argument should not be optional.